wagtail/wagtailadmin/templates/wagtailadmin/shared/breadcrumb.html

```diff
@@ -1,8 +1,10 @@
 {% load i18n wagtailadmin_tags %}
 
 <ul class="breadcrumb">
-    {% for page in page.get_ancestors %}
-        {% if page.is_root %}
+    {% explorable_ancestors page request.user as ancestors %}
+    {% for page in ancestors %}
+        {% is_explorable_root page request.user as page_is_explorable_root %}
+        {% if page_is_explorable_root %}
```

Comment:
What should I see in breadcrumbs if:
* I have tree like this:

   ```
   <Root>
   |- Home page
       |- Blog index
          |- Blog post 1
          |- Blog post 2
          |- Blog post 3
              |- Blog post 3 - page 1
              |- Blog post 3 - page 2
       |- Some other page
   ```
* I have all permissions for "Blog index"
* I will open "Blog post 3 - page 1"?

I see only one item with `icon-home` and i'm afraid that I got not an expected result :)